test/unit_tests/arkode/CXX_serial/ark_test_butcher.cpp

```diff
@@ -123,12 +90,13 @@ int main()
 
   // loop over individual DIRK tables
   std::cout << "\nTesting individual DIRK methods:\n\n";
-  for (std::string table : Tables_DIRK)
+  for (int table = ARKODE_MIN_DIRK_NUM; table <= ARKODE_MAX_DIRK_NUM; table++)
   {
     std::cout << "Testing method " << table << ":";
```

Comment:
I like that this file is somewhat more maintainable as we add new Butcher tables to ARKODE.  However, one thing that I particularly liked about the existing version is that the output displayed the table name itself, not an integer value.  Since there is no simple/fast way to look up the name from a given enum value (I do not consider having to open `arkode_butcher_erk.h` and manually count lines inside the enum as either simple or fast), this update obfuscates the connection between method performance and the name itself.

Reply:
Yeah the output is not as informative. The function `arkButcherTableERKNameToID` does allow us to convert an ID to a name. I'll check if that can be accessed here to keep the original output style.

Reply:
I agree the output is not ideal.

> Since there is no simple/fast way to look up the name from a given enum value

I can create the reserve function of [`arkButcherTableERKNameToID`](https://github.com/LLNL/sundials/blob/b685654c6958699821fc68a822b66076fec8cae7/src/arkode/arkode_butcher_erk.c#L62). Then we could map the id from the loop to its name and have the original output style.